demos-europe/demosplan-core
# refactor(refs DPLAN-24): allow dragging an faq item into another category [vue 3]

### Ticket
https://demoseurope.youtrack.cloud/issue/DPLAN-24/VueDraggable-durch-vue3-kompatible-Alternative-ersetzen




In the FAQ List (role: Redaktion) we now allow dragging an item into another category.

### How to review/test

- log in as role "Redaktion"
- go to /faq/verwalten
- drag an item inside a category and make sure it is saved at the correct position
- drag an item to another category and make sure it is saved at the correct position

### Linked PRs (optional)
- [ ] https://github.com/demos-europe/demosplan-ui/pull/682

### Tasks
- [ ] select correct target branch
- [ ] allow saving item in another category in the BE

### PR Checklist
